### DISCUSSION TITLE
Contact info?

### DISCUSSION THREAD

**Discussion by JohnHaTrick** [LanceGundersen/navfit2020#112] [Q&A] [answered]
Hi @LanceGundersen, there's a Capt Schwerin who'd like to ask you about this project.

Is lance.gundersen.ctr@navy.mil a good email for you?

Not sure if GitHub has private messages... sorry if this is too public.

**Comment by LanceGundersen** [comment, marked as answer]
Hey @JohnHaTrick have them reach out to my personal email gundersen.lance@gmail.com as it's checked regularly.
